DigitalLearningSolutions.Data.Tests/TestHelpers/SearchSortFilterAndPaginateTestHelper.cs

```diff
@@ -30,7 +30,8 @@ ISearchSortFilterPaginateService searchSortFilterPaginateService
                             options!.PaginationOptions?.PageNumber ?? 1,
                             1,
                             options.PaginationOptions?.ItemsPerPage ?? int.MaxValue,
-                            items.Count()
+                            items.Count,
+                            items.Count <= 250

```

Comment:
Do we want to make the 250 a parameter with a default value of 250 on the off chance this gets used in the future for something?